lib/order_form.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'takeaway'
+require_relative 'menu'
+
```

Comment:
This object is doing too much, so to refactor we'd split out this class into smaller classes each with a single responsibility - this one would just handle the order (you could maybe just call it order as well)